File: C++/Smallest_largest.cpp
#include<iostream>
using namespace std;
int smallest(int a, int b, int c){
    int min=a;
    if(b<min){
        min=b;
    }
    if(c<min){
        min=c;
    }
    return min;
}
int largest(int a, int b, int c){
    int max=a;
    if(b>max){
        max=b;
    }
    if(c>max){
        max=c;
    }
    return max;
}
int main(){
    int a,b,c;
    cout<<"Enter 3 numbers :";
    cin>>a>>b>>c;
    cout<<"Largest : "<<largest(a,b,c)<<endl;
    cout<<"Smallest: "<<smallest(a,b,c);
    return 0;
}

File: Java/Smallest_largest.java
package com.lev;

import java.util.Scanner;

public class Smallest_largest {
    public static int smallest(int a, int b, int c){
        int sm = a;
        if(b < sm){
            sm = b;
        }
        if(c < sm){
            sm = c;
        }
        return sm;
    }
    public static int largest(int a, int b, int c){
        int lg = a;
        if(b > lg){
            lg = b;
        }
        if(c > lg){
            lg = c;
        }
        return lg;
    }

    public static void main(String[] args) {
        System.out.print("Enter 3 numbers : ");
        Scanner in = new Scanner(System.in);
        int a = in.nextInt();
        int b = in.nextInt();
        int c = in.nextInt();
        int sm = smallest(a,b,c);
        int lg = largest(a,b,c);
        System.out.println("Largest : "+lg);
        System.out.println("Smallest : "+sm);
    }
}
